tests/resources/test_resources.py

```diff
@@ -65,11 +58,27 @@ def test_simple_flow(mock_apply_async, app, db, client, user):
     expected_job["updated"] = res.json["updated"]
     assert res.json == expected_job
 
+    list_repr = deepcopy(expected_job)
+    list_repr.update(
+        {
+            "last_run": {"queue": "celery", "title": "Manual run"},
```

Comment:
why is this returned? As I understand, you just activated the job but didn't run...Also, `last_run` shows the actual last run while `last_runs` shows the history of runs groups by status?

Reply:
discussed IRL